### ISSUE TITLE
Export the YDocUpdatedEvent type

### ISSUE THREAD

**Issue by edbartholomew** [liveblocks/liveblocks#1726]
**What is the improvement or update you wish to see?**

I'd like the `YDocUpdatedEvent` type to be exported. 

As far as I can see the following types are exported from `node_modules/@liveblocks/node/dist/index.d.mts`

```
export { type CommentCreatedEvent, type CommentDeletedEvent, type CommentEditedEvent, type CommentReactionAdded, type CommentReactionRemoved, Liveblocks, LiveblocksError, type LiveblocksOptions, type NotificationEvent, type RoomAccesses, type RoomCreatedEvent, type RoomData, type RoomDeletedEvent, type RoomInfo, type RoomPermission, type RoomUser, type Schema, type StorageUpdatedEvent, type ThreadCreatedEvent, type ThreadMetadataUpdatedEvent, type ThreadParticipants, type UserEnteredEvent, type UserLeftEvent, type WebhookEvent, WebhookHandler, type WebhookRequest };
```

I'm using @liveblocks/node: v2.0.2

**Comment by nvie** [comment]
These types should indeed be exported at the top-level. This is fixed in #1728, and will be published later today as 2.0.3. Thanks for reporting!

**Comment by edbartholomew** [comment]
Awesome thanks 🙏

**Comment by nvie** [comment]
This fix was published yesterday in [v2.0.3](https://github.com/liveblocks/liveblocks/releases/tag/v2.0.3).
